### PR TITLE
✨ File based logs for self-hosted

### PR DESCRIPTION
**Pull request by RTLS**

### CODE REVIEW

**Comment by RTLS** [comment]
* **#388** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/388?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @RTLS and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/sequin/logs/logs.ex [hunk 1]
```diff
@@ -7,12 +7,13 @@ defmodule Sequin.Logs do
 
   defp env, do: Application.fetch_env!(:sequin, :env)
   defp self_hosted, do: Application.fetch_env!(:sequin, :self_hosted)
+  defp datadog_enabled, do: env() in [:dev, :test] or self_hosted()
```
Comment (acco) [inline lib/sequin/logs/logs.ex:10]: Just occurred we could maybe even boil this down to "is datadog configured?"

Then we'd immediately get "hey you can use datadog in self hosted" for free

File: lib/sequin/logs/rotate_logs_worker.ex [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  require Logger
+
+  defp env, do: Application.fetch_env!(:sequin, :env)
+  defp self_hosted, do: Application.fetch_env!(:sequin, :self_hosted)
+
+  @impl Oban.Worker
+  def perform(_) do
+    Logger.info("[RotateLogsWorker] Running")
+
+    if env() in [:dev] or self_hosted() do
```
Comment (acco) [inline lib/sequin/logs/rotate_logs_worker.ex:16]: We can have this call `Logs.log_to_file?` or whatever, which can then check if DD enabled

**Review by acco — APPROVED** [review]
Carter, see minor comments

**Pull request merged**